gum/backend-dbghelp/gumsymbolutil-dbghelp.c

```diff
@@ -148,6 +152,36 @@ gum_find_functions_matching (const gchar * str)
   return matches;
 }
 
+gboolean
+gum_load_symbols (const gchar * path)
+{
+  GumDbghelpImpl * dbghelp;
+  DWORD64 base;
+  WCHAR * path_utf16;
+  gboolean success;
+
+  dbghelp = gum_dbghelp_impl_try_obtain ();
+  if (dbghelp == NULL)
+    return FALSE;
+
+  path_utf16 = (WCHAR *) g_utf8_to_utf16 (path, -1, NULL, NULL, NULL);
+  base = GetModuleHandleW (path_utf16);
```

Comment:
```suggestion
  base = GetModuleHandleW (path_utf16);
  if (base == 0)
    goto beach;
```

- We might want to bail early here if this fails, as we don't want to give `SymLoadModuleExW` a base of zero.
- Do we need a cast here? Seem to recall that `HMODULE` is a pointer type but might be misremembering.